src/components/Pivot/Pivot.md

```diff
@@ -0,0 +1,43 @@
+# Pivot
+Includes a Persona component with contact options.
+
+## Variants
+
+### Default
+@@include('Pivot.html')
+
+### Large
+@@include('Pivot.Large.html')
+
+### Tabs
```

Comment:
OK - I'd say we go on ahead and pull this in but flag it is an issue for Design to loop back on and provide us with the documentation.